website/content/api-docs/secret/pki.mdx

```diff
@@ -142,24 +144,30 @@ $ certbot certonly --server https://localhost:8200/v1/pki/acme/directory ...
-in the table), behavior is specified by the `default_directory_policy` in the
-[ACME configuration](#set-acme-configuration).  These directories can also be
-forbidden by setting that policy as `forbid`.  If the policy is `sign-verbatim`
-then _any_ identifier for which the client can prove ownership of will be
-issued for.  This is similar to using the [Sign Verbatim](#sign-verbatim)
-endpoint, but with additional verification that the client has proven
-ownership (within the ACME protocol) of the requested certificate
-identifiers.
+| Method | Path                                                               | Default Directory Policy   | Issuer                | Role          |
+|:-------|:-------------------------------------------------------------------|:---------------------------|:----------------------|:--------------|
+| `ACME` | `/pki/acme/directory`                                              | `sign-verbatim`            | `default`             | Sign-Verbatim |
+| `ACME` | `/pki/acme/directory`                                              | `role:role_ref`            | Specified by the role | `:role_ref`   |
+| `ACME` | `/pki/acme/directory`                                              | `external-policy(:policy)` | Specified by CIEPS    | CIEPS         |
+| `ACME` | `/pki/issuer/:issuer_ref/acme/directory`                           | `sign-verbatim`            | `:issuer_ref`         | Sign-Verbatim |
+| `ACME` | `/pki/issuer/:issuer_ref/acme/directory`                           | `role:role_ref`            | `:issuer_ref`         | `:role_ref`   |
+| `ACME` | `/pki/roles/:role/acme/directory`                                  | (any)                      | Specified by the role | `:role`       |
+| `ACME` | `/pki/issuer/:issuer_ref/roles/:role/acme/directory`               | (any)                      | `:issuer_ref`         | `:role`       |
+| `ACME` | `/pki/external-policy(/:policy)/acme/directory`                    | (any)                      | Specified by CIEPS    | CIEPS         |
+| `ACME` | `/pki/issuer/:issuer_ref/external-policy(/:policy)/acme/directory` | (any)                      | Specified by CIEPS    | CIEPS         |
+
+When a role is not explicitly specified, behavior is specified by the
+`default_directory_policy` in the [ACME configuration](#set-acme-configuration).
+These directories can also be forbidden by setting that policy as `forbid`.  If
+the policy is `sign-verbatim` then _any_ identifier for which the client can
+prove ownership of will be issued for.  This is similar to using the
+[Sign Verbatim](#sign-verbatim) endpoint, but with additional verification
+that the client has proven ownership (within the ACME protocol) of the
+requested certificate identifiers. When `external-policy` is specified as the
+default value, the CIEPS engine <EnterpriseAlert inline="true" /> is used for
+validating and templating the certificate. An optional policy name can be
```

Comment:
This kinda sounds like ACME validation is not used, maybe add `is also used for validating and templating the certificate ` ?

Reply:
Does this read better?

```
When `external-policy` is specified as the
default value, the CIEPS engine <EnterpriseAlert inline="true" /> is used for
validating and templating the certificate instead of a role; ACME's challenge
validation is still enforced.
```

This clarifies that CIEPS is equivalent in this case to the role restrictions, which also do not replace ACME's challenge validation.

Reply:
Yup, at least to me.